crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -35,21 +50,56 @@ pub const MESH_SHADER_HANDLE: HandleUntyped =
 
 impl Plugin for MeshRenderPlugin {
     fn build(&self, app: &mut bevy_app::App) {
-        let mut shaders = app.world.get_resource_mut::<Assets<Shader>>().unwrap();
-        shaders.set_untracked(
-            MESH_SHADER_HANDLE,
-            Shader::from_wgsl(include_str!("mesh.wgsl")),
-        );
-        shaders.set_untracked(
-            MESH_STRUCT_HANDLE,
-            Shader::from_wgsl(include_str!("mesh_struct.wgsl"))
+        #[cfg(not(feature = "bevy_shader_hot_reloading"))]
+        {
+            let mut shaders = app.world.get_resource_mut::<Assets<Shader>>().unwrap();
+            shaders.set_untracked(
+                MESH_SHADER_HANDLE,
+                Shader::from_wgsl(include_str!(
+                    "../../../../assets/shaders/bevy_pbr/mesh.wgsl"
+                )),
+            );
+            shaders.set_untracked(
+                MESH_STRUCT_HANDLE,
+                Shader::from_wgsl(include_str!(
+                    "../../../../assets/shaders/bevy_pbr/mesh_struct.wgsl"
+                ))
                 .with_import_path("bevy_pbr::mesh_struct"),
-        );
-        shaders.set_untracked(
-            MESH_VIEW_BIND_GROUP_HANDLE,
-            Shader::from_wgsl(include_str!("mesh_view_bind_group.wgsl"))
+            );
+            shaders.set_untracked(
+                MESH_VIEW_BIND_GROUP_HANDLE,
+                Shader::from_wgsl(include_str!(
+                    "../../../../assets/shaders/bevy_pbr/mesh_view_bind_group.wgsl"
+                ))
                 .with_import_path("bevy_pbr::mesh_view_bind_group"),
```

Comment:
Perhaps add constants for these import paths, since they are now repeated? (to avoid accidentally renaming one and forgetting about the other)

Reply:
Good idea. Done.